src/environments/environment.prod.ts

```diff
@@ -1,6 +1,7 @@
 export const environment = {
     production: true,
     googleAnalyticsKey: 'UA-124734893-3',
+    googleTagManagerKey: 'GTM-TQ7PKDF',
```

Comment:
Have you confirmed this is the key for MO? Seems to be the same as MC, that's why I ask.